src/test/java/hmwk16.java

```diff
@@ -0,0 +1,55 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+
+public class hmwk16 {
```

Comment:
Recommendation: class names should be nouns, in mixed cases with the first letter of each internal word capitalized.

for example

`public class Hmwk16`